src/Apps/Order/Utils/getErrorDialogCopy.tsx

```diff
@@ -0,0 +1,80 @@
+import { RouterLink } from "System/Router/RouterLink"
+import React from "react"
+
+const SUPPORT_EMAIL = "orders@artsy.net"
+
+export enum ErrorDialogs {
+  CurrencyNotSupported = "currency_not_supported",
+}
+
+interface ErrorDialogCopy {
+  title: string
+  message: string
+  formattedMessage: React.ReactNode
+}
+
+export const getErrorDialogCopy = (dialog?: ErrorDialogs): ErrorDialogCopy => {
+  switch (dialog) {
+    case ErrorDialogs.CurrencyNotSupported:
+      return currencyNotSupportedErrorDialogCopy()
+    default:
+      return defaultErrorDialogCopy()
+  }
+}
+
+const currencyNotSupportedErrorDialogCopy = () => {
+  const title = "Payment declined"
+  const message = `This card is not compatible with the currency for this artwork. Please confirm with your card issuer if this currency can be supported, try a different payment method or contact ${SUPPORT_EMAIL}.`
+
+  return errorDialogCopy(title, message)
+}
+
+const defaultErrorDialogCopy = () => {
+  const title = "An error occurred"
+  const message = `Something went wrong. Please try again or contact ${SUPPORT_EMAIL}.`
+
+  return errorDialogCopy(title, message)
+}
+
+const errorDialogCopy = (title: string, message: string): ErrorDialogCopy => {
+  return {
+    title,
+    message,
+    formattedMessage: formatErrorDialogMessage(message),
+  }
+}
+
+export const formatErrorDialogMessage = (message: string) => {
```

Comment:
This is a component, so it should be `ComponentCase`

Reply:
I thought I was working with a utility function until you pointed out that it's a component. This feels a lot better. Thank you.

Reply:
Yah, out of habit i always surface this when I see it, because I cant count the number of time i've confused a complex interpolated variable `{foo}` for something simple, and then later found myself in the weeds. If it clearly renders JSX (`<Foo />`) you'll always know at a glance that something significant might be inside.